src/sparseml/pytorch/torch_to_onnx_exporter.py

```diff
@@ -190,6 +190,22 @@ def transform(
             )
         }
 
+        # adjust 4-bit ranges to 8-bits, since ONNX does not have 4-bit support
+        fake_quant_modules = [
+            m for m in module.modules() if m.__class__.__name__ == "FakeQuantize"
+        ]
+        for quant in fake_quant_modules:
+            # original ranges preserved in quant.quant_min and quant.quant_max
+            quant_range = (
+                quant.activation_post_process.quant_min,
+                quant.activation_post_process.quant_max,
+            )
+            if quant_range == (-8, 7):  # convert int4 range to int8
+                quant.activation_post_process.quant_min = -128
+                quant.activation_post_process.quant_max = 127
+            elif quant_range == (0, 15):  # convert uint4 to uint8
+                quant.activation_post_process.quant_max = 255
```

Comment:
to support arbitrary bit widths, lets support any quant range that isn't int8 or uint8 (choosing int vs uint depending on if `quant_min < 0`)